### PR TITLE
Display correct number of tasks in MissionQueueCard

### PR DESCRIPTION
**Pull request by UsamaEquinorAFK**

### CODE REVIEW

File: frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueCard.tsx [hunk 1]
```diff
@@ -120,7 +120,7 @@ export const MissionQueueCard = ({ order, mission, onDeleteMission }: MissionQue
     const [confirmDeleteDialogOpen, setConfirmDeleteDialogOpen] = useState<boolean>(false)
     const fillColor = tokens.colors.infographic.primary__energy_red_21.hex
     let numberOfTasks = 0
-    mission.tasks.forEach((task) => (numberOfTasks += task.inspections.length))
+    mission.tasks.forEach((task) => (numberOfTasks += 1))
```
Comment (Eddasol) [inline frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueCard.tsx:123]: just use tasks.length?

Comment (UsamaEquinorAFK) [reply frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueCard.tsx:123]: Nope- that is a loop my solution gives correct answer. You can test both ways.

Comment (UsamaEquinorAFK) [reply frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueCard.tsx]: Removed loop and assigned const directly now =)

**Review by Eddasol — APPROVED** [review]
LGTM :)

**Pull request merged**